k8s/start.sh

```diff
@@ -2,13 +2,15 @@
 
 ########################################################################
 # This script starts the k8s daemonset
-# You must
-# a) enter your scalyr write key
-# b) edit configmap.yaml to include any environment-aware variables
+# 
+# Instructions:
+# a) set your Scalyr write key in your shell ( e.g. export SCALYR_API_KEY=<YOUR_SCALYR_KEY> )
+# b) edit configmap.yaml to set cluster name (to match the k8s cluster you want to monitor)
+# c) source ./start.sh
 ########################################################################
 
 kubectl delete secret scalyr-api-key
-kubectl create secret generic scalyr-api-key --from-literal=scalyr-api-key=YOUR_WRITE_KEY
+kubectl create secret generic scalyr-api-key --from-literal=scalyr-api-key=${SCALYR_API_KEY}
```

Comment:
You know, the "right way" to do this is to first delete the scalyr agent daemonset.. then the config map.. then the secret key.. then the service account.. and then create everything in the reverse order of that.. create service account, create secret key, create config map, create daemonset.

I'm debating if it's worthwhile to make that change now (just because it is a last minute change).. however, if you run it a few times and works, I think we are fine..